### DISCUSSION TITLE
Where is apk?

### DISCUSSION THREAD

**Discussion by jachin84** [dotnet/dotnet-docker#4989] [General]
I am using a `mcr.microsoft.com/dotnet/sdk:6.0` as a base for a custom build container that has a few extra things installed, such as the azure artifacts credentials provider. I would also like to add the zip package but I can't find `apk`. Am I going nuts, I can't see it anywhere. It's available in alpine:3.18 but not in the sdk image.

**Comment by omajid** [comment]
The `mcr.microsoft.com/dotnet/sdk:6.0` container image is built on top of Debian. If you are looking for alpine, maybe you want `sdk:6.0-alpine`?

```
$ podman run -it mcr.microsoft.com/dotnet/sdk:6.0 cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
$ podman run -it mcr.microsoft.com/dotnet/sdk:6.0-alpine cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.18.4
PRETTY_NAME="Alpine Linux v3.18"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
```

**Comment by stambaugh101** [comment]
I feel your frustration! It's puzzling that the zip package is missing in mcr.microsoft.com/dotnet/sdk:6.0. Have you tried checking the package manager for the specific SDK version? If not, it might be worth [PC](https://topfollowapks.net/topfollow-for-pc/) exploring that avenue to ensure you're not overlooking any dependencies. Good luck with your custom build container!

**Comment by jachin84** [comment]
All good, my mistake. I got my package managers mixed up between Linux distros.
